.storybook/preview.js

```diff
@@ -1,11 +1,15 @@
 import GlobalStyles from "../src/GlobalStyles"
+import { MemoryRouter } from "react-router";
 
 export const decorators = [
   (Story) => (
     <>
     <GlobalStyles />
     <Story />
     </>
+  ),
+  (Story) => (
+    <MemoryRouter><Story/></MemoryRouter>
   )
 ]
```

Comment:
You could to it together:
```suggestion
export const decorators = [
  (Story) => (
    <MemoryRouter>
      <GlobalStyles />
      <Story />
    </MemoryRouter>
  )
]
```